analyses/doublet-detection/README.md

```diff
@@ -22,17 +22,32 @@ conda-lock install --name openscpca-doublet-detection conda-lock.yml
 conda activate openscpca-doublet-detection
 ```
 
-Then, run the following bash script:
+There are two portions of this module, which can be run as follows:
+
+- Benchmarking analysis: Compare several doublet detection methods on publicly available (non-ScPCA) datasets with annotated doublets.
+This analysis runs several doublet detection methods and assesses and compares their performance.
+  ```sh
+  ./run_doublet-detection-benchmarking.sh
+  ```
+
+- ScPCA analysis: Detect doublets in ScPCA data using `scDblFinder` for a given project id (`SCPCPXXXXXX`).
+This analysis exports a TSV file for each library in a given ScPCA project with `scDblFinder` inferences.
```

Comment:
Can you expand on what columns we should expect in the TSV?

Reply:
This might belong in the output files section, but either way you should describe the contents of the TSV file.